code/modules/projectiles/gun.dm

```diff
@@ -46,7 +46,7 @@
 /obj/item/weapon/gun/examine(mob/user)
 	..()
 	if(pin)
-		user << "It has a [pin] installed."
+		user << "It has [pin] installed."
```

Comment:
Capitalized item names aren't standardized, and would causes issues elsewhere as the game would consider it a Proper Noun.